internal/controller/driver_controller.go

```diff
@@ -327,6 +328,61 @@ func (r *driverReconcile) LoadAndValidateDesiredState() error {
 	return nil
 }
 
+func (r *driverReconcile) reconcileLogRotateConfigMap() error {
+	logRotateConfigmap := &corev1.ConfigMap{}
+	logRotateConfigmap.Name = utils.LogRotateConfigMapName(r.driver.Name)
+	logRotateConfigmap.Namespace = r.driver.Namespace
+
+	log := r.log.WithValues("logRotateConfigMap", logRotateConfigmap.Name)
+	log.Info("Reconciling logRotate configmap")
+
+	logRotationSpec := cmp.Or(r.driver.Spec.Log, &csiv1a1.LogSpec{}).Rotation
+	if logRotationSpec != nil {
+		opResult, err := ctrlutil.CreateOrUpdate(r.ctx, r.Client, logRotateConfigmap, func() error {
+			if _, err := utils.ToggleOwnerReference(true, logRotateConfigmap, &r.driver, r.Scheme); err != nil {
+				log.Error(err, "Failed adding an owner reference on the LogRotate config map")
+				return err
+			}
+			if logRotationSpec.Periodicity != "" && logRotationSpec.MaxLogSize.String() == "0" {
+				return fmt.Errorf("maxLogSize must be set when periodicity is set")
+			}
+			maxFiles := cmp.Or(logRotationSpec.MaxFiles, defaultLogRotateMaxFiles)
+			cronLogRotateSettings := []string{
+				"\tmisslingok",
+				"\tcompress",
+				"\tcopytruncate",
+				"\tnotifempty",
+				fmt.Sprintf("\trotate %d", maxFiles),
+			}
+			if logRotationSpec.Periodicity != "" {
+				periodicity := string(logRotationSpec.Periodicity)
+				cronLogRotateSettings = append(cronLogRotateSettings, "\t"+periodicity)
```

Comment:
To align with the rest of the code, let's have the "\t" be precatenated as part of the value construction and not part of the append operation. 
```suggestion
				periodicity := "\t"+string(logRotationSpec.Periodicity)
				cronLogRotateSettings = append(cronLogRotateSettings, periodicity)
```